.github/workflows/ci_common.yml

```diff
@@ -1081,3 +1077,27 @@ jobs:
           echo "Checking tag2-${{ github.run_id }} exists... "
           echo "$versions" | grep -q "tag1-${{ github.run_id }}"
           echo "Tag exists."
+
+  test-github-output-job1:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      output1: ${{ steps.github-output.outputs.test }}
+      output2: ${{ steps.github-output.outputs.test2 }}
+    steps:
+      - id: github-output
+        uses: ./
+        with:
+          runCmd: |
```

Comment:
Could have used `run: echo "test=hello" >> "$GITHUB_OUTPUT"` instead, but kept with runArg per other cases. 'uses: ./` may be incorrect

Reply:
`uses: ./` tells the GH workflow to run the step using the devcontainer/ci action. We reference it by path here so that we pick up the code from the PR branch rather than a published version of the action